pkg/agent/protocols/manager.go

```diff
@@ -0,0 +1,15 @@
+package protocols
+
+type ProtocolManager interface {
+	Download(modelDir string, storageUri string) error
+}
+
+type Protocol string
+
+const (
+	S3    Protocol = "s3://"
+	//GCS    Protocol = "gs://"
+	//PVC   Protocol = "pvc://"
+	//File  Protocol = "file://"
+	//HTTPS Protocol = "https://"
+)
```

Comment:
You editor should be adding newlines on save here.

Reply:
Yeah, I hadn't ran `fmt vet` yet :)